### DISCUSSION TITLE
wipe while retraction

### DISCUSSION THREAD

**Discussion by Khronos666** [OrcaSlicer/OrcaSlicer#5270] [General]
faced with such a problem, maybe
"wipe while retraction" does not work properly... Or is that how it should be?

![Desktop Screenshot 2024 05 05 - 12 25 48 84](https://github.com/SoftFever/OrcaSlicer/assets/123404675/27ad3a77-9ef9-49a1-b7c2-853c466f4939)
![Desktop Screenshot 2024 05 05 - 12 25 35 73](https://github.com/SoftFever/OrcaSlicer/assets/123404675/c2af0aa7-22d3-4feb-8b24-8e9c9e9a556e)

**Comment by vgdh** [comment]
It's not clear what do you mean.

**Reply by Khronos666** [reply]
When "wipe while retraction" is enabled, cleaning occurs only with a random seam. And with other seam options, cleaning does not occur, only in places...in the screenshots you can see, from the white dot, the beginning of the seam, when closing, there is a wipe, a yellow line.
I would like to have a wipe on each seam, thereby closing the seam.
